mongo/integration/unified/event.go

```diff
@@ -29,7 +31,7 @@ const (
 	connectionCheckOutFailedEvent   monitoringEventType = "ConnectionCheckOutFailedEvent"
 	connectionCheckedOutEvent       monitoringEventType = "ConnectionCheckedOutEvent"
 	connectionCheckedInEvent        monitoringEventType = "ConnectionCheckedInEvent"
-	serverDescriptionChangedEvent   monitoringEventType = "ServerDescriptionChangedEvent"
+	serverDescriptionChangedInfo    monitoringEventType = "ServerDescriptionChangedEvent"
```

Comment:
Resurrecting this comment:
This rename creates inconsistent naming that breaks reader's expectation. It would be much clearer to name the new `serverDescriptionChangedEvent` type to something else (e.g. `serverDescriptionChangedInfo`).